s3torchconnectorclient/rust/src/exception.rs

```diff
@@ -15,6 +15,10 @@ pyo3::create_exception!(
     PyException
 );
 
+fn log_error(message: &str) {
+    println!("ERROR: {}", message);
```

Comment:
This won't go through the rust logging we've got set up: https://github.com/awslabs/s3-connector-for-pytorch/blob/main/s3torchconnectorclient/rust/src/logger_setup.rs

Reply:
I have addressed this, added error!() log